### ISSUE TITLE
Could you split fit-html.tx into another npm package?

### ISSUE THREAD

**Issue by harryfei** [Festify/fit-html#26]
So we can use fit-html's awesome `withFit` function to write shared web components without any `redux` dependency.

**Comment by NeoLegends** [comment]
Splitting of a separate package shouldn't be necessary for this. Just importing the `withFit`-mixin directly will be enough. :)

**Comment by harryfei** [comment]
Ok, I got it. That's cool.

**Comment by harryfei** [comment]
BTW, is `3Kb` still correct now? I just get a 20Kb bundle file(fit-html, lit-html, and redux) and an 8Kb compressed file.

**Comment by NeoLegends** [comment]
> BTW, is 3Kb still correct now? I just get a 20Kb bundle file(fit-html, lit-html, and redux) and an 8Kb compressed file. @NeoLegends

3k was always minzipped size, but that sounds a little large, tbh. fit-html hasn't changed in size very much lately, but lit-html and maybe redux have. How many components are there in your bundle? You may underestimate the size of the template strings. Also, are you using a tree-shaking bundler (webpack 2, rollup)?

**Comment by harryfei** [comment]
Yeah, I check the bundle file, it's not `fit-html`'s fault. `lit-html` increase a lot in 0.10 version.

**Comment by harryfei** [comment]
I write the index.ts and import lit-html (v0.10), use rollup to build bundle.

``` typescript
import { html } from 'lit-html/lib/lit-extended';
console.log(html)
```
minized size: 5.2Kb
minized and gizped size: 1.9Kb

**Comment by NeoLegends** [comment]
I'm afraid you're missing the `render` from `lit-html` to get real stats about the size. Right now you're just importing the tag, but no render functionality.
